models/hh_cond_beta_gap_traub.h

```diff
@@ -89,19 +89,18 @@ to match those used with NEST 1.9.10 when preparing data for [1]_. Code for all
 simulators covered is available from ModelDB [3]_.
 
 Note:
-In this model, a spike is emitted if
-
- :math:`V_m >= V_T + 30 mV` and `V_m` has fallen during the current time step
+In this model, a spike is emitted if :math:`V_m \geq V_T + 30` mV and :math:`V_m` has fallen during the current time
+step.
```

Comment:
I'd prefer if lines were broken at a reasonable length to ease reading also in the code files.

Reply:
All the new or changed lines are wrapped at 120 characters, should we keep wrapping at 80 characters in the RST blocks to keep them internally consistent?

Reply:
Exactly!

Reply:
Also, because `Alt`+`Q` does it like this in Emacs ;-)

Reply:
The line length debate is not dead yet! (Will it ever?) Let's save this for a rainy day.

Reformatting: done!